diff --git a/py/libs/api/stability.py b/py/libs/api/stability.py
index 3b40db2..f17a45a 100644
--- a/py/libs/api/stability.py
+++ b/py/libs/api/stability.py
@@ -10,7 +10,7 @@ from ..log import log_node_error
 
 
 root_path = pathlib.Path(__file__).parent.parent.parent.parent
-config_path = os.path.join(root_path,'config.yaml')
+config_path = os.path.join(root_path,'config','config.yaml')
 default_key = [{'name':'Default', 'key':''}]
 
 
